Changelog.md

```diff
@@ -7,7 +7,7 @@ Language Features:
 
 Compiler Features:
  * Yul Optimizer: Remove ``mstore`` and ``sstore`` operations if the slot already contains the same value.
-
+ * Commandline Interface: Event and error signatures are also returned when using `--hashes`.
```

Comment:
```suggestion
 * Commandline Interface: Event and error signatures are also returned when using ``--hashes``.
```

Single quotes work too but for some reason we're always using double ones in the changelog and I'd stick to that.